commands/src/main/java/org/wildfly/extras/creaper/commands/messaging/AddAcceptor.java

```diff
@@ -0,0 +1,72 @@
+package org.wildfly.extras.creaper.commands.messaging;
+
+/**
+ * Command which creates and adds an acceptor to given server
+ */
+public final class AddAcceptor extends AbstractTransportConfigAddCommand {
+
+    private AddAcceptor(InVmBuilder builder) {
+        super(builder);
+    }
+
+    private AddAcceptor(GenericBuilder builder) {
+        super(builder);
+    }
+
+    private AddAcceptor(RemoteBuilder builder) {
+        super(builder);
+    }
+
+    @Override
+    protected TransportConfigItem getConfigType() {
+        return TransportConfigItem.ACCEPTOR;
+    }
+
+    public static final class InVmBuilder extends AbstractTransportConfigAddCommand.InVmBuilder<InVmBuilder> {
+
+        public InVmBuilder(String name) {
+            super(name);
+        }
+
+        public InVmBuilder(String name, String serverName) {
+            super(name, serverName);
+        }
+
+        @Override
+        public AddAcceptor build() {
+            return new AddAcceptor(validate());
+        }
+    }
+
+    public static final class RemoteBuilder extends AbstractTransportConfigAddCommand.RemoteBuilder<RemoteBuilder> {
+
+        public RemoteBuilder(String name) {
+            super(name);
+        }
+
+        public RemoteBuilder(String name, String serverName) {
+            super(name, serverName);
+        }
+
+        @Override
+        public AddAcceptor build() {
+            return new AddAcceptor(this);
```

Comment:
In `InVmBuilder`, you `return new AddAcceptor(validate())`, but here, you `return new AddAcceptor(this)`. How come? (My suggestion would be to make `validate` return `void` and call it in all `build` methods before the `return` statement.)